### ISSUE TITLE
Passport 8.x compatibility

### ISSUE THREAD

**Issue by ArondeParon** [ipunkt/laravel-oauth-introspection#12]
The current version is compatible up to 7.x. Installing this dependency on a setup with passport 8.x will yield errors.

**Comment by MattApril** [comment]
For anyone interested, I've added Passport 8.x support to my own fork here: https://github.com/MattApril/laravel-oauth-introspection/commit/3fd70f7611e63990fc8c468f89cfd8a8ab86250f

**Comment by rokde** [comment]
@MattApril do you want to make pull request?

**Comment by MattApril** [comment]
Yes, I can find some time today hopefully. I did not think this was still
maintained.

**Comment by ghost** [comment]
I suppose Passport 10.x compatibility also yields errors, of course.

**Comment by ghost** [comment]
Laravel Passport 10.x compatibility seems to be working with some modifications found in this fork:

https://github.com/userh-dev/laravel-oauth-introspection

Changes from @MattApril were also ported.
